robottelo/cli/factory.py

```diff
@@ -391,6 +392,46 @@ def make_gpg_key(options=None):
     return create_object(GPGKey, args, options)
 
 
+@cacheable
+def make_content_credential(options=None):
+    """Creates a content credential.
+
+    In Satellite 6.8 only gpg_key is supported.
+
+    :param options: Check options using `hammer content-credentials create --help` on satellite.
+
+    :returns GPGKey object
```

Comment:
Does it? Not "`ContentCredential` object"?

Reply:
Yes, thanks for spotting that:
`:returns ContentCredential object`